fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -3957,7 +3958,8 @@ public static boolean canUseNereidsDistributePlanner() {
             // TODO: support other sink
             if (!(logicalPlan instanceof UnboundResultSink
                     || logicalPlan instanceof LogicalFileSink
-                    || logicalPlan instanceof InsertIntoTableCommand)) {
+                    || logicalPlan instanceof InsertIntoTableCommand
+                    || logicalPlan instanceof UnboundBlackholeSink)) {
```

Comment:
please support NereidsDistributePlanner, the legacy coordinator will be removed later